datasets/offenseval_dravidian/README.md

```diff
@@ -184,6 +184,22 @@ This work is licensed under a [Creative Commons Attribution 4.0 International Li
 
 ### Citation Information
 
+
+```
+@article{chakravarthi-etal-2021-lre,
+title = "DravidianCodeMix: Sentiment Analysis and Offensive Language Identification Dataset for Dravidian Languages in Code-Mixed Text",
+author = "Chakravarthi, Bharathi Raja  and
+  Priyadharshini, Ruba  and
+  Muralidaran, Vigneshwaran and
+  Jose, Navya and
+  Suryawanshi, Shardul and
+  Sherly, Elizabeth  and
+  McCrae, John P",
+  journal={Language Resources and Evaluation},
+  publisher={Springer}
+}
+
+```
```

Comment:
This one looks incomplete, is this normal ?

Reply:
Hi @lhoestq, thanks for reviewing the PR.
As the manuscript is under review in the Language Resources and Evaluation Journal, which could take months before it gets published online.  Hence, it's a common practice to have a citation like this. I hope this helps.

Reply:
makes sense thanks !